### PR TITLE
Update Pages “sponsors”

### PR DESCRIPTION
**Pull request by kristinedill**
Automatically generated by Netlify CMS

### CODE REVIEW

File: _pages/sponsors.md
Comment (kylekirkby) [inline _pages/sponsors.md:30]: @kristinedill - if you're using this `feature_block` section, you'll need and image. Jekyll is failing to build the site since this image path is missing.

**Review by kylekirkby — CHANGES_REQUESTED** [review]

File: _pages/sponsors.md
Comment (kristinedill) [reply _pages/sponsors.md:30]: Thanks, Kyle. I'll get a new image and try adding it.

**Comment by kristinedill** [comment]
Hi @kylekirkby the content on this page that I added in Netlify is not showing up on the preview. Also, I updated the header image with the new LVC21F one and I think it's the wrong size or resolution, but not sure how to tell. 

Also, the same link is still broken https://www.linaro.org/contact/irc/ so it's failing and I'm not sure how to fix that since it's a Linaro page. 
Can you help with this? 

Thanks, 
Kristine

**Comment by kylekirkby** [comment]
> Hi @kylekirkby the content on this page that I added in Netlify is not showing up on the preview. Also, I updated the header image with the new LVC21F one and I think it's the wrong size or resolution, but not sure how to tell.
> 
> Also, the same link is still broken https://www.linaro.org/contact/irc/ so it's failing and I'm not sure how to fix that since it's a Linaro page.
> Can you help with this?
> 
> Thanks,
> Kristine

Hi @kristinedill,

I've updated the page in NetlifyCMS. If you're using a slider, the title will need to be in the slider section.

The section was in the wrong format. It needed to be `main_content_row`. That will then display any content you've added in the "main content" section.

I'll take a look at the image. 

Cheers,

Kyle

**Comment by kylekirkby** [comment]
@kristinedill - you may need to refresh the page.

https://production-connect-linaro-org-834.ghactions.linaro.org/sponsors/

**Comment by kristinedill** [comment]
Hi @EbbaSimpson  or @kylekirkby  can you approve these changes?

Thanks! 

Kristine

**Review by EbbaSimpson — APPROVED** [review]

**Comment by kristinedill** [comment]
@kylekirkby @EbbaSimpson not sure why but I can't merge this even though it's been approved by Ebba.

"Merging is blocked
Merging can be performed automatically once the requested changes are addressed."

**Comment by kylekirkby** [comment]
@kristinedill - please try again. I had requested changes but the request was not marked as resolved.

**Comment by kristinedill** [comment]
Hi @kylekirkby how do I approve requested changes in the future? 

I tried to publish this just now via Netlify and got an error message: failed to publish. 2 of 2 required status checks are required. at least 1 approving review is required.

Thanks

**Review by kylekirkby — APPROVED** [review]

**Comment by kylekirkby** [comment]
> Hi @kylekirkby how do I approve requested changes in the future?
> 
> I tried to publish this just now via Netlify and got an error message: failed to publish. 2 of 2 required status checks are required. at least 1 approving review is required.
> 
> Thanks

@kristinedill please try now. It looks like if I have requested changes, I have to approve that the requested changes have been made. It's rare that changes are requested but if I do request them in the future, I'll make sure to approve once it's been resolved.

**Pull request merged**
